spec/models/like_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+	describe 'default likes' do
+		let(:like) { build :like }
+
+		context "Like validations :" do
+			it 'should return true when like is valid' do
+		   		expect(like.valid?).to be true
+		  	end
+
+		  	it 'should return false when like is invalid becaouse of empty user' do
```

Comment:
Let's use another variable to make this more intuitive:
`expect(invalid_like).to_not be_valid`